### ISSUE TITLE
Publish Documenso on Public Cloud Marketplace (GCP/AWS)

### ISSUE THREAD

**Issue by daallgeier** [documenso/documenso#197]
### Is your proposal related to a problem?

As Independent Software Providers and Resellers we work with Google Cloud Platform, Amazon Web Services, and others to enable our customers to take advantage of applications on their respective infrastructures and through their marketplaces.  

### Describe the solution you'd like

Publish documenso on Cloud Marketplaces with Public and Private Pricing.  See also how GitLab is doing it: https://about.gitlab.com/partners/technology-partners/google-cloud-platform/

### Describe alternatives you've considered

Help customers deploy on GCP through terraform with the open source code (and no kickback to documenso) 

### Additional context

https://about.gitlab.com/handbook/support/license-and-renewals/workflows/self-managed/gcp_marketplace_license.html

**Comment by daallgeier** [comment]
Yeah terraform is going to be the first important step and it definitely needs to be continued to be supported for many reasons.  
If you ever need help with the GCP partnership specifically, let me know. I have good insights into this process by now.

**Comment by ElTimuro** [comment]
Planned but no timeline
